ui/lib/apps/Statement/pages/List/index.tsx

```diff
@@ -47,8 +65,44 @@ export default function StatementsOverview() {
     allStmtTypes,
     loadingStatements,
     tableColumns,
+
+    genDownloadToken,
+    downloading,
   } = controller
 
+  async function exportCSV() {
+    message.info(t('statement.pages.overview.toolbar.exporting') + '...', 2)
+    const token = await genDownloadToken()
+    if (token) {
+      const url = `${client.getBasePath()}/statements/download?token=${token}`
+      // `window.open(url)` would cause browser popup interception if genDownloadToken takes long time
+      // window.open(url)
+      downloadByLink(url)
+    }
+  }
+
+  function menuItemClick({ key }) {
+    switch (key) {
+      case 'settings':
+        setShowSettings(true)
+        break
+      case 'export':
+        exportCSV()
+        break
+    }
+  }
+
+  const dropdownMenu = (
+    <Menu onClick={menuItemClick}>
+      <Menu.Item key="settings">{t('statement.settings.title')}</Menu.Item>
```

Comment:
Recommend to add some icons to make it prettier :)

Reply:
Done.